lib/master.js

```diff
@@ -23,7 +23,9 @@ function liveWorkerIds() {
       worker.process.exitCode == null;
     return isAlive;
   });
-  return workerIds;
+  return workerIds.sort(function(a, b) {
```

Comment:
I guess what I'm suggesting is to add a comment complaining about the behaviour of `Array.prototype.sort` and how it sorts numbers as strings and then either use the sort you have or use lodash to shorten it by a couple lines.